commands/hook.go

```diff
@@ -35,7 +35,7 @@ func executeHook(hookName string) error {
 	}
 
 	// 3. Execute Husky scripts
-	err = executeScriptsInDir(filepath.Join(".husky", hookName))
+	err = executeHuskyGitHook(filepath.Join(".husky/_", hookName))
```

Comment:
While we don't really care about OS compatibility at the moment, it's a good practise to let  `filepath.Join` handle all the path operators.

```suggestion
	err = executeHuskyGitHook(filepath.Join(".husky", "_", hookName))
```
https://pkg.go.dev/path/filepath#Join
> Join joins any number of path elements into a single path, separating them with an OS specific [Separator](https://pkg.go.dev/path/filepath#Separator). Empty elements are ignored. The result is Cleaned.

> The filepath package uses either forward slashes or backslashes